examples/enforce-gatekeeper/warning-only/README.md

```diff
@@ -2,17 +2,40 @@
 
 ### Overview
 
-This example is very similar to the invalid configmap example. The major
+This example is very similar to the invalid configmap example. It also
```

Comment:
Please audit all the docs for go/kpt-docs-style-guide

e.g. style issue: configmap -> `ConfigMap`